### DISCUSSION TITLE
有人開始用chatgpt看論文了嗎?

### DISCUSSION THREAD

**Discussion by DuxtinTai** [1101-GS4719-C/Python-Community-GS4719-C#1204] [General]
最近我正進行一個電機領域的專題研究，發現閱讀相關論文是一個相當耗時的工作，尤其是因為電機領域的論文通常以英文書寫，所以需要花費更多的時間和精力來熟悉專有詞彙和結構，已因此我嘗試使用 ChatGPT 來幫助我閱讀論文；在使用了一段時間後，我發現 ChatGPT 提供了一些方便的功能，讓我更加容易理解論文的內容，但同時也有些問題需要注意。

首先， ChatGPT 可以生成文章摘要，這讓我在開始閱讀正文之前可以有個整體的認識。尤其是當我剛開始接觸一個新領域時，摘要對我來說非常有用，它讓我能夠快速熟悉相關主題，了解研究目標和方法，也讓我能夠更加高效地閱讀全文。

其次， ChatGPT 可以幫助我回答一些簡單的問題，例如概括一個論文的主旨或是解釋某個專有詞彙的意思。這樣我就不需要在翻譯或是查詢某些詞彙的意思上花費太多時間，而可以更加專注於閱讀論文的內容。

但是需要注意的是， ChatGPT 在處理結構化資訊時可能會有一些缺陷，例如表格或圖片的資訊可能會被誤解。所以，當需要深入研究某些細節時，我還是需要自己閱讀全文，以便更好地理解論文的內容。

總體來說， ChatGPT 提供了一些方便的功能，讓我在閱讀論文時更加高效，但它僅能提供相對簡要的資訊，無法取代人工閱讀。因此，我會建議大家在使用 ChatGPT 的同時仍需保持警覺，避免誤解和錯誤的理解。

以上是我的看法，希望能拋磚引玉，歡迎大家在下面幫忙補充相關的經驗喔~

**Comment by lyien1021** [comment]
我有！目前看了2篇論文，真心覺得沒有ChatGPT我大概一篇要看一個月吧...而且ChatGPT不僅能解釋文章中很多專有名詞，在他閱讀完整篇文章後還能解釋前後文的關係，雖然它也是有講錯的時候啦，但整體而言真的大大增加看論文的效率。

**Reply by DuxtinTai** [reply]
同學，你提到chatgpt可以幫助你理解前後文，想向你請益你都如何問問題呢？像是怎麼打關鍵字？
現在問問題的技能真的越來越重要了，有些國外大學也開始注重這一塊的能力了。

**Reply by lyien1021** [reply]
我是先把文章內容輸入ChatGPT後，可以問他兩段之間的關聯是什麼？或是像這段為什麼要提到某個東西？也可以讓他解釋圖表所代表的意義。

**Comment by HsuSungTing** [comment]
有的，不過個人的經驗是，chatgpt可以幫我們快速建構一個論文的whole picture
但細部的一些理解還是要找到更好的辦法弄懂........
個人有被教授打槍的經驗，但我也還不知道到底是我問chatgpt的問題不對，還是我自己理解錯，還是chatgpt講的東西是錯的.......

**Reply by DuxtinTai** [reply]
我也覺得chatgpt細部的問題，目前處理不是很好。
如果從全文摘要開始，不斷往細部問題問下去，有時候就會出現一些明顯的錯誤。當遇到chatgpt不清楚的問題，chagpt不會說不明白，而是會回覆一些錯誤的內容。
同時，糾正chatgpt錯誤過後也有可能繼續有錯誤內容出現，這點真的是要特別注意。

**Comment by shuxian12** [comment]
市面上應該已經有很多專門的paper summarizer，或許可以先使用專門的summarizer去對paper做統整
如果有遇到不懂的專有名詞，可能再向chatGPT做詢問會比較好，畢竟chatGPT主要還是對話的機器人，多少會有些些錯誤
另外一方面也可以請chatGPT給其他專有名詞的例子，在之後做詢問或回答的時候，他的準確度會比較高喔！！

**Comment by curtis1017** [comment]
ChatGPT在解釋專有名詞上非常的精準，網路上常常有許多對同一個專有名詞，卻有不同的解釋，但是ChatGPT會經過他的分析後，再來給你精確的解釋，還有一個好處，那就是如果有不懂的，他一定都會經過你給他的要求來做修正。

**Reply by sjh501** [reply]
但我常常遇到每次生成的解釋都不相同反而讓我很混淆，只能說ChatGPT在一些部分的確還有待加強。

**Comment by orianna901001** [comment]
身為常常在寫英文報告的學生，我覺得雖然Chatgpt在一定程度上能給出大概的輪廓，但我相信教授真正會喜歡的是更具體、更聚焦的東西，而非只是chatgpt寫出來的summary。

**Comment by Jack-1D** [comment]
chatgpt確實是個不錯的工具，撇除可能的人為介入，只要chatgpt能中立的把文章摘要出來，確實能省下不少時間。可是我認為有時候chatgpt不一定能輸出我們想要的重點，那可能又要花額外的時間重看文章。

**Comment by a3240281370** [comment]
就像大家說的，chatgpt根據訓練所得的知識回答關於特定主題的問題，並提供一般性的資訊和觀點。如果有關於特定論文或學術領域的問題，主要還是提供相關的背景知識和解答，但無法提供對該論文的具體分析或詳細引用。

**Comment by sjh501** [comment]
我第一次使用ChatGPT時也被它的統整功能驚豔到，最近也有開始使用ChatGPT快速瀏覽和統整一些論文。感謝這篇文章讓我知道在使用ChatGPT看論文時有什麼要注意的地方，少走很多冤枉路。

**Comment by Superlan8** [comment]
就我的經驗，現在太多人都只依賴chatgpt，只要是它給的答案就輕易相信，完全沒有去查證或是驗證，完全把chatgpt當神在用，而且目前我們大部分人使用的都是gpt3.5的版本，這個版本只訓練了2021以前的資料，像我最近看一些新paper，chatgpt根本在亂回答，甚至你可能每次問它，它給的答案都不一樣，我認為使用chatgpt應該還是要以輔助為主，不要全然相信了。

**Comment by Alexgns** [comment]
在已經有基本概念的情況下，chatgpt確實能夠加速論文閱讀，但有時未必提供完全正確的資訊，因此若要仰賴chatgpt，也並需要有判斷其是否有誤的能力
